### PR TITLE
add patch for PyTorch 1.2.0 to fix failing softmax test on Intel Sandy Bridge

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bb2c1a09.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/2b292f42141ddc9dc3a1551948a672d0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8e8902d0f9254e97717befbb518ba982 for a full test report.

**Comment by branfosj** [comment]
Thanks for sorting this out.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/602e33b016abd3cc2663c8ff2f3e4567 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2410.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1b023ad4307192f8106f1ae1edd34bd2 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!
